Gemfile.lock

```diff
@@ -48,32 +47,33 @@ GEM
     method_source (0.9.2)
     mini_portile2 (2.4.0)
     minitest (5.11.3)
-    money (6.13.2)
+    money (6.13.4)
       i18n (>= 0.6.4, <= 2)
-    nokogiri (1.10.1)
+    nokogiri (1.10.4)
       mini_portile2 (~> 2.4.0)
     pagerduty (2.1.2)
       json (>= 1.7.7)
-    rack (2.0.6)
+    rack (2.0.7)
     rack-test (1.1.0)
       rack (>= 1.0, < 3)
     rails-dom-testing (2.0.3)
       activesupport (>= 4.2.0)
       nokogiri (>= 1.6)
-    rails-html-sanitizer (1.0.4)
+    rails-html-sanitizer (1.2.0)
       loofah (~> 2.2, >= 2.2.2)
-    railties (5.2.2.1)
-      actionpack (= 5.2.2.1)
-      activesupport (= 5.2.2.1)
+    railties (6.0.0)
```

Comment:
### [railties](https://rubyonrails.org)

**Major** version upgrade :chart_with_upwards_trend::exclamation: 5.2.2.1 → 6.0.0

[_[change-log](https://github.com/rails/rails/blob/v6.0.0/railties/CHANGELOG.md), [source-code](https://github.com/rails/rails/tree/v6.0.0/railties)_]





<details>
<summary>Commits</summary>

A change of **7275** commits. See the full changes on [the compare page](https://github.com/rails/rails/compare/v5.2.2.1...v6.0.0).

These are the first 10 commits:
- (045b307) [Merge pull request #36159 from sharang-d/update-getting-star…](https://github.com/rails/rails/commit/045b307cf3de9a4629f3f7bd574e671399f48872)
- (18e684b) [Merge pull request #36152 from prathamesh-sonpatki/active-su…](https://github.com/rails/rails/commit/18e684b5ec9de0f460c8b215c208702c2b2e2e28)
- (fa94115) [Merge pull request #36166 from alexcameron89/add_action_cabl…](https://github.com/rails/rails/commit/fa94115a3d2bd7ed7f0284750fbf47238f7969e3)
- (d846e30) [Merge pull request #36172 from tgxworld/recover_pluck_perf](https://github.com/rails/rails/commit/d846e30380f1b1c2731f427480005df6b799790b)
- (485de3d) [Merge pull request #36169 from prathamesh-sonpatki/backport-…](https://github.com/rails/rails/commit/485de3d9f2d7164ece1b0cc283bb406af670ff10)
- (72999db) [Make generated test work even when using virtual attributes](https://github.com/rails/rails/commit/72999db353bf262a9e675f2e7dd24f20f401b596)
- (ffefb6d) [Merge pull request #36190 from kamipo/should_attempt_tx_call…](https://github.com/rails/rails/commit/ffefb6d42880a4e60b0f12085afe16a1b9a6569d)
- (7b00308) [Fix a bug where DebugExceptions errors out when malformed qu…](https://github.com/rails/rails/commit/7b00308161241ac93df707f7ec84bfe6ed46cf80)
- (d0a7460) [Merge pull request #36196 from st0012/fix-29947](https://github.com/rails/rails/commit/d0a746047e03b63e70c85d284a23d2aee3d5057f)
- (5768921) [documents autoloading in the upgrading guide [ci skip]](https://github.com/rails/rails/commit/57689218c92f8a320b339bf645518259c25fbb1d)


</details>